build/crds/edgecluster/mission_v1.yaml

```diff
@@ -29,8 +29,19 @@ spec:
          spec:
           type: object
           properties:
-            content: 
+            missionresource:
               type: string
+            missioncommand:
+              type: object
+              properties:
+                trigger:
+                  type: string
+                runiftriggersucceed:
```

Comment:
this boolean == true means command below will run once trigger event happens?

what does it mean what this boolean is set to false? always run?

runiftriggersucceed --> runwhentriggered ?

Reply:
I renamed to runwhentriggersucceed